panel/tests/ui/layout/test_card.py

```diff
@@ -70,10 +70,10 @@ def test_card_not_collapsible(page, port, card_components):
     serve_panel_widget(page, port, card)
 
     # no card button to disable collapsing the card
-    card_button = page.locator('.bk.card-button')
+    card_button = page.locator('.card-button')
     expect(card_button).to_have_count(0)
     # card header and other inner objects
-    card_elements = page.locator('.bk.card > .bk')
+    card_elements = page.locator('.card > .bk')
```

Comment:
The other `.bk` needs to be dropped as well, but I'm not sure what to suggest in place of it.

Reply:
Yes. This PR right now is basically just a big search and replace.